dmcontent/html.py

```diff
@@ -100,7 +106,10 @@ def text_to_html(
         value = filters.capitalize_first(value)
 
     if format_links is True:
-        value = filters.format_links(value)
+        if open_links_in_new_tab is True:
+            value = filters.format_links(value, open_links_in_new_tab)
+        else:
+            value = filters.format_links(value)
```

Comment:
We can simplify this to 
```suggestion
        value = filters.format_links(value, open_links_in_new_tab)
```

Reply:
Ha, so we can!